templates/solutions/ai/edge.html

```diff
@@ -0,0 +1,272 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/17eHoiLrhFLrfV8UXD_4bxn-yY1_YToxf4cs8-5kYkxo/edit
+{% endblock meta_copydoc %}
+
+{% block title %}Enterprise AI edge computing{% endblock %}
+
+{% block meta_description %}
+  Get everything you need to deploy AI at the edge in one solution. Secure and compliant open source optimised for embedded AI use cases.
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-suru--50-50 p-section--shallow">
+    <div class="row--50-50">
+      <div class="col">
+        <h1 class="p-section--shallow">
+          Develop AI models and run
+          <br />
+          them at the edge &mdash; all on the
```

Comment:
```suggestion
          them at the edge &mdash; all on the
```
Usually in UK English we opt for the ndash, but not a biggie

Reply:
I'm never confident about these tbh, but if this is norm in the UK then I'll use this from now on 🙂